.renovaterc.json

```diff
@@ -2,6 +2,7 @@
     "extends": [
       ":gitSignOff",
       "group:recommended",
+      ":pinAllExceptPeerDependencies",
```

Comment:
I am not sure if this is the right fix, we are not using this config preset in loopback-next (see https://github.com/strongloop/loopback-next/blob/master/.renovaterc.json):

```
  "extends": [
    ":gitSignOff",
    "group:monorepos",
    "group:recommended",
    "helpers:disableTypesNodeMajor"
  ],
```